src/validate/methods/xdr.ts

```diff
@@ -0,0 +1,7 @@
+export const xdr = (value: string) => {
```

Comment:
should we name this to `validateBase64`? In our current lab, we have two methods: [validateBase64](https://github.com/stellar/laboratory/blob/ed2094d980bdeb967e5c636ea6053e31ab9c8773/src/helpers/validateBase64.ts#L4) and [validateTxXdr](https://github.com/stellar/laboratory/blob/ed2094d980bdeb967e5c636ea6053e31ab9c8773/src/helpers/validateTxXdr.ts#L5)